source/standards/testing-services.html.md.erb

```diff
@@ -0,0 +1,69 @@
+---
+title: Testing Services
+expires: 2018-01-10
+---
+
+# <%= current_page.data.title %>
+
+<%= partial :expires %>
+
+## Why and when to test
+
+You should carry out [penetration testing](https://www.gov.uk/service-manual/technology/vulnerability-and-penetration-testing) and vulnerability assessments to ensure your service is secure, according to the [Service Manual](https://www.gov.uk/service-manual):
+
+> "Vulnerability assessments help you find potential weaknesses in your service.
+> Penetration tests proactively attack your systems to find weaknesses and help you understand how easy they are to exploit."
+
+At a minimum you should test twice a year at six month intervals, your tests should consist of:
+
+* one external test by an [approved third party](https://www.gov.uk/service-manual/technology/vulnerability-and-penetration-testing#working-with-third-parties) which is classed as a major test
+* one internal test which is classed as a minor test
```

Comment:
We don't seem to mention who can carry out this internal test? Maybe:

>  * one internal test which is classed as a minor test carried out by members of the GDS Security Team

or 

>  * one internal test which is classed as a minor test carried out by our internal ethical hackers

Reply:
Done